src/test/java/uk/gov/hmcts/ccd/domain/service/jsonpath/CaseDetailsJsonParserTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.ccd.domain.service.jsonpath;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.ccd.config.JacksonUtils;
+import uk.gov.hmcts.ccd.data.documentdata.DocumentData;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.hmcts.ccd.config.JacksonUtils.MAPPER;
+
+class CaseDetailsJsonParserTest {
```

Comment:
Think we may be missing some test scenarios in this class - there are a number more `public` methods. Some can probably be made `private` on first glance, but can't see any tests on `containsDocumentUrl()`, for example.